functional-tests/fn_el_bridge_precompile.py

```diff
@@ -23,9 +25,10 @@
 class ElBridgePrecompileTest(flexitest.Test):
     def __init__(self, ctx: flexitest.InitContext):
         ctx.set_env("basic")
+        self.logger = logging.getLogger(Path(__file__).stem)
 
     def main(self, ctx: flexitest.RunContext):
-        print("SKIPPING TEST fn_el_bridge_precompile")
+        self.logger.debug("SKIPPING TEST fn_el_bridge_precompile")
```

Comment:
This should be an `info` no?
```suggestion
        self.logger.info("SKIPPING TEST fn_el_bridge_precompile")
```

Reply:
Actually it should be a warning, just so it jumps out at us.  Although this whole test can probably be removed now since we have a end-to-end withdrawal test, right?

Reply:
Yes. Happy path only since that's what currently supported in devnet.